src/System.Data.DataSetExtensions/tests/Mono/DataTableExtensionsTest.cs

```diff
@@ -40,6 +40,8 @@ namespace MonoTests.System.Data
 {
     public class DataTableExtensionsTest
     {
+        private string _testDataSet = "testdataset1.xml";
```

Comment:
Why do we need these changes?

Reply:
Right now the way that we bin place and run the tests is through CopyToOutputDirectory and through the build output to the OutputDirectory, the way that the build bin places items is with the same path with the same structure as we have it in the test project. (i.e if we have a txt file under `test-project-folder/myFiles/text.txt`it will be bin placed under a myFiles directory in the output directory). 

With this new changes, we would bin place the working directory through the bin place targets and this bin places all the dependencies under the same folder without subdirectories, so I needed to update the tests that reference files to an specific path under the working directory, to reference this files to the root working directory.